src/lib/stores/wallet/network.ts

```diff
@@ -100,6 +107,11 @@ const NETWORK_TOKENS: ValueForEachSupportedChain<string> = {
   [314]: 'FIL',
 };
 
+const NETWORK_AUTO_WRAPPED_PAIRS: Partial<ValueForEachSupportedChain<AutoUnwrapPair[] | []>> = {
+  [1]: [{ name: 'Ethereum', nativeSymbol: 'ETH', wrappedSymbol: 'WETH' }],
+  [314]: [{ name: 'Filecoin', nativeSymbol: 'FIL', wrappedSymbol: 'WFIL' }],
```

Comment:
Should these go into the `additional-tokens.json` instead?

Reply:
I think here is fine. I'd rather not add to additional-tokens bc the "token list" standard is explicitly only for ERC-20s AFAIK